elixir-phoenix-deployement.md

```diff
@@ -0,0 +1,162 @@
+
+
+## PaaS vs VPS
+
+In the past we have mostly used Platform as a Service (PaaS) providers
+to deploy Elixir applications. The main advantage of PaaS
+is it is quick to setup and have your application running
+for your users.
+However in the longer term using a PaaS can become costly (per unit of compute e.g. RAM/CPU).
+Virtual Private Server are in contrast normally cheaper to run
+but require more setup time and DevOps knowledge to maintain.
+
+### PaaS
+
+#### Heroku
+
+We are used to deploy Elixir/Phoenix on Heroku
+and we have already a ["how to"](https://github.com/dwyl/learn-phoenix-framework/blob/master/heroku-deployment.md) guide of this process.
+
+However there are some limitations when using Heroku with Phoenix (see https://hexdocs.pm/phoenix/heroku.html):
+![image](https://user-images.githubusercontent.com/6057298/83642784-2e1d1200-a5a7-11ea-8c97-9c7dd920469c.png)
+
+pricing:
+![image](https://user-images.githubusercontent.com/6057298/83641712-cc0fdd00-a5a5-11ea-8cbf-49981ae3747e.png)
+
+The pricing above doesn't include the database which needs to be added to the total cost, see https://elements.heroku.com/addons/heroku-postgresql
+
+
+### Gigalixir
+
+Similar to Heroku [Gigalixir](https://www.gigalixir.com/) provides
+a platform which focuses _exclusively_ on deploying Elixir/Phoenix application.
+Having an exclusive focus on deploying Elixir Apps gives Gigalixir several key advantages:
+1. Zero-downtime blue-green continuous deployment.
+see: https://martinfowler.com/bliki/BlueGreenDeployment.html
+2. No limit to concurrent connections.
+3. All clustering handled transparently. 
+(so if you need to scale your app beyond 10k concurrent users, 
+you don't have to pay exponentially more for bigger "dynos" 
+the way you are forced to scale vertically on Heroku)
+
+Deploying: https://elixircasts.io/deploying-with-gigalixir-%28revised%29
+
+Pricing:
+
+![image](https://user-images.githubusercontent.com/6057298/83643629-290c9280-a5a8-11ea-89da-648daec204f3.png)
+
+Gigalixir vs Heroku:
+
+![image](https://user-images.githubusercontent.com/6057298/83644108-d1225b80-a5a8-11ea-869a-c8c0e4a28012.png)
+
+Like Heroku you need to add the cost for using the database:
+
+![image](https://user-images.githubusercontent.com/6057298/83644296-09c23500-a5a9-11ea-9fdd-c50532195c95.png)
+
+see also tiers pricing page: https://gigalixir.readthedocs.io/en/latest/tiers-pricing.html
+
+
+### Render
+
+[Render](https://render.com/) is another Paas similar to Heroku
+
+pricing:
+
+![image](https://user-images.githubusercontent.com/6057298/83645052-e9df4100-a5a9-11ea-8d3e-f1b18bb4cc02.png)
+
+
+
+## Virtual Private Server
+
+VPS allow us to manage ourself the deployement
+setup. This allow us to customise the server and
+the costs linked to it.
+
+
+### Linode
+
+#### Ubuntu
+
+Linode provides and support Ubuntu:
+![image](https://user-images.githubusercontent.com/6057298/83647811-3bd59600-a5ad-11ea-893c-b99e3df7f605.png)
```

Comment:
```suggestion
![linode-distribution-options-screen](https://user-images.githubusercontent.com/6057298/83647811-3bd59600-a5ad-11ea-893c-b99e3df7f605.png)
```

@SimonLab please get into the habit of _naming_ your images so that it's easier to review/maintain the docs. 😉 
(_and if you see me **not** naming my images, please point it out and/or fix it! thanks!_)